apps/server/src/modules/user/service/user.service.spec.ts

```diff
@@ -851,6 +851,8 @@ describe('UserService', () => {
 				userRepo.findByIdOrNull.mockResolvedValueOnce(user);
 				userRepo.deleteUser.mockResolvedValue(1);
 
+				config.get.mockReturnValue(true);
```

Comment:
maybe specify which config get should be mocked instead of a general mock return?

Reply:
done